src/renderers/shaders/ShaderChunk/bsdfs.glsl

```diff
@@ -0,0 +1,152 @@
+//#define ENERGY_PRESERVING_MONOCHROME
+
+float calcLightAttenuation( float lightDistance, float cutoffDistance, float decayExponent ) {
+
+	if ( decayExponent > 0.0 ) {
+
+	  return pow( saturate( -lightDistance / cutoffDistance + 1.0 ), decayExponent );
+
+	}
+
+	return 1.0;
+
+}
+
+
+void BRDF_Lambert( const in IncidentLight incidentLight, const in GeometricContext geometryContext, const in vec3 diffuseColor, inout ReflectedLight reflectedLight ) {
+
+	float lambertianReflectance = saturate( dot( geometryContext.normal, incidentLight.direction ) );
+
+	#if defined( ENERGY_PRESERVING_MONOCHROME ) || defined( ENERGY_PRESERVING_RGB )
+
+		lambertianReflectance *= RECIPROCAL_PI;
```

Comment:
Actually this is address a bug in our existing code.  Basically lambertian reflectance is a absorption of the incoming light by the surface and then an emission of that light in a uniform distribution in the hemisphere above the surface modulated by the surface color.  Thus the incoming light = outgoing light * diffuseColor.  But the issue is that incoming light in from a single direction, the outgoing light is in a hemisphere, thus if you are perceiving that outgoing light from a certain direction you only get 1 / PI of the total outgoing light.  If you try to integrate our current outgoing diffuse light across the hemisphere, you'll notice that it is emitting more light that it is receiving.

I added it to the ENERGY_PERSERVING modes to retain backwards compatibility, but this is the correct way to do Lambertian reflectance.